doc/syntax.rst

```diff
@@ -444,7 +444,7 @@ Using type-specific formatting::
 
    auto t = tm();
    t.tm_year = 2010 - 1900;
-   t.tm_mon = 6;
+   t.tm_mon = 8;
```

Comment:
I think it should be 7 because tm_mon starts from 0: https://godbolt.org/z/b9fv81

Reply:
Yeah, you're right.